### PR TITLE
Bump to llvm/torch-mlir@061bbc5e1bc4f7880bb565e404a6709f97396818

### PR DESCRIPTION
**Pull request by rsuderman**
Bumping torch-mlir forwards to head

### CODE REVIEW

**Comment by ScottTodd** [comment]
This might include some CPU regressions like `onnx/node/generated/test_pow_types_int32_int32` as on https://github.com/iree-org/iree/pull/19430, FYI. IDK where that ended up... the discussion thread went cold.

File: tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json [hunk 1]
```diff
@@ -120,6 +120,14 @@
...
     "onnx/node/generated/test_nonmaxsuppression_two_classes",
     "onnx/node/generated/test_nonzero_example",
+    "onnx/node/generated/test_pow_types_float32_int32",
+    "onnx/node/generated/test_pow_types_float32_int64",
+    "onnx/node/generated/test_pow_types_float32_uint32",
+    "onnx/node/generated/test_pow_types_float32_uint64",
+    "onnx/node/generated/test_pow_types_int32_float32",
+    "onnx/node/generated/test_pow_types_int32_int32",
+    "onnx/node/generated/test_pow_types_int64_float32",
+    "onnx/node/generated/test_pow_types_int64_int64",
```
Comment (ScottTodd) [inline tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json:130]: Not comfortable with these regressions, but I guess we can proceed to keep things moving.

File: third_party/torch-mlir
Comment (ScottTodd) [inline third_party/torch-mlir:1]: Nice, three new model tests passing thanks to https://github.com/iree-org/torch-mlir/commit/19949835400392b71fe8f04760d1fa4855beab70

Before https://github.com/iree-org/iree/actions/runs/12417121729/job/34667922529#step:8:132:
```
ERROR    onnx_models.conftest:conftest.py:139 iree-compile stderr:
ERROR    onnx_models.conftest:conftest.py:140 artifacts/vision/classification/rcnn-ilsvrc13-9_version17.mlir:13:82: error: expected '>'
    %10 = torch.operator "onnx.Constant"() {torch.onnx.value = dense_resource<_fc-rcnn_b_0> : tensor<200xf32>} : () -> !torch.vtensor<[200],f32> 
                                                                                 ^
XFAIL                                                                    [ 58%]
iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_resnet50_v1 
```

After https://github.com/iree-org/iree/actions/runs/12419685503/job/34675893696?pr=19531#step:8:375:
```
XFAIL iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_inception_v1
XPASS iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_efficientnet_lite4
XPASS iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_mobilenet
XPASS iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_rcnn_ilsvrc13
XPASS iree-test-suites/onnx_models/tests/vision/classification_models_test.py::test_zfnet_512
============= 12 passed, 1 xfailed, 4 xpassed in 169.86s (0:02:49) =============
```

(mobilenet xpass is already recorded with https://github.com/iree-org/iree-test-suites/commit/2187b6fc8717300f9bc4292b4bb1468736bb77d0)

**Review by ScottTodd — APPROVED** [review]

File: tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json (hunk 1 above)
Comment (rsuderman) [reply tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json:130]: There is no value in halting the roll for more than a few days. If we see regressions we need someone to grab and author a fix.

Comment (ScottTodd) [reply tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json:130]: Tried that, got no response :/

https://github.com/iree-org/iree/pull/19430#issuecomment-2542560879

**Pull request merged**